types/net_fields.d.ts

```diff
@@ -77,6 +77,10 @@ type NetStartsUsingFields = Fields & {
   targetId: 6;
   target: 7;
   castTime: 8;
+  x: 9;
+  y: 10;
+  z: 11;
+  heading: 12;
```

Comment:
Thanks for catching this.  I thought there was something like this file, but my brain was like "maybe that was just in some iteration of raidemulator typescriptization that got removed" because I couldn't find it.